lib/client/workflow-client/workflowClient.js

```diff
@@ -155,7 +155,7 @@ WorkflowMediatorService.prototype.checkStatus = function checkStatus(workorder,
     status = CONSTANTS.STATUS.COMPLETE_DISPLAY;
   } else if (!workorder.assignee) {
     status = CONSTANTS.STATUS.UNASSIGNED_DISPLAY;
-  } else if (stepReview.nextStepIndex < 0) {
+  } else if (!result) {
```

Comment:
nextStepIndex is used to identify the next step the workflow should move to.

`stepReview.complete` flags the workflow as being complete.

nextStepIndex can be 0 and if there is a result, it is considered "In Progress"
nextStepIndex can be 0 and it there is no result, it should be considered "New"